app/src/controllers/electrificationProject.ts

```diff
@@ -74,11 +75,7 @@ const controller = {
     next: NextFunction
   ) => {
     try {
-      const { electrificationProject } = await controller.generateElectrificationProjectData(
-        req.body,
-        IntakeStatus.SUBMITTED,
-        req.currentContext
-      );
+      const electrificationProject = await controller.generateElectrificationProjectData(req.body, req.currentContext);
```

Comment:
Note: With Express v4 to v5 update we'll need to check the req.body here to make sure it's not undefined. (if navs are gonna be creating empty elec. projects like housing projs)